google/cloud/ndb/context.py

```diff
@@ -60,43 +61,30 @@ def __next__(self):
 _context_ids = _ContextIds()
 
 
-try:  # pragma: NO PY2 COVER
-    import contextvars
+class _LocalState:
+    """Thread local state."""
 
-    class _LocalState:
-        """Thread local state."""
-
-        def __init__(self):
-            self._toplevel_context = contextvars.ContextVar(
-                "_toplevel_context", default=None
-            )
-            self._context = contextvars.ContextVar("_context", default=None)
-
-        @property
-        def context(self):
-            return self._context.get()
-
-        @context.setter
-        def context(self, value):
-            self._context.set(value)
-
-        @property
-        def toplevel_context(self):
-            return self._toplevel_context.get()
-
-        @toplevel_context.setter
-        def toplevel_context(self, value):
-            self._toplevel_context.set(value)
+    def __init__(self):
+        self._toplevel_context = contextvars.ContextVar(
+            "_toplevel_context", default=None
+        )
+        self._context = contextvars.ContextVar("_context", default=None)
 
+    @property
+    def context(self):
+        return self._context.get()
 
-except ImportError:  # pragma: NO PY3 COVER
```

Comment:
Note: it's easier to see that this change is correct if you view it with split-diffs and while ignoring whitespace changes